CONTRIBUTING.md

```diff
@@ -50,11 +50,31 @@ include BREAKING CHANGE keyword for breaking changes.
 
 The message summary should be a one-sentence description of the change, and it must be 72 characters in length or shorter. For a list of tags, please [click here](https://github.com/conventional-changelog/commitlint/tree/master/%40commitlint/config-conventional#type-enum). We use the angular style for tags. See the [default release rules](https://github.com/semantic-release/commit-analyzer/blob/master/lib/default-release-rules.js) based on the commit tag. Shown below is an example of the release type that will be done based on a commit message.
 
-| Commit message                                                                                                                                          | Release type               |
-| ------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------- |
-| `fix(preview): fullscreen behavior in ie11`                                                                                                             | Patch Release              |
-| `feat(preview): add ability to fullscreen`                                                                                                              | ~~Minor~~ Feature Release  |
-| `feat(preview): remove fullscreen ability`<br><br>`BREAKING CHANGE: The fullscreen ability has been removed due to poor support of the fullscreen api.` | ~~Major~~ Breaking Release |
+#### Semantic Versioning
+
+"Semantic release" means that changes to the version number of the package (e.g. `3.42.11` to `3.43.0`) are done according to rules that indicate how the change will affect consumers.
+
+The version number is broken into 3 positions &mdash; `Major.Minor.Patch`. In semantic release terms, changes to the numbers follow `Breaking.Feature.Fix`.
+
+- Anything that changes or removes an API, option, or output format is a `BREAKING CHANGE`.
+- Anything that adds new functionality in a backwards-compatible way is a feature (`feat`). Consumers have to upgrade to the new version to use the feature, but nothing will break if they do so.
+- Bugfixes (`fix`) for existing behavior are a patch. Consumers don't have to do anything but upgrade.
+  - Performance fixes (`perf`) and reverts (`revert`) are treated as patch releases.
+  - Automated commits from Box's internalization team cause a patch because they will use the format `fix(moji): ***`.
+- Other prefixes, such as `docs`, don't get released and don't appear in the changelog. These tags signal that there are **no external changes to _any_ APIs** (including non-breaking ones).
+  Changes from these types of commits will get released along with the next releasable commit.
```

Comment:
What does this mean? -> `next releasable commit`.

Reply:
next feat/fix/perf commit

Reply:
>  Changes from these types of commits will get released only when the release script detects other releasable commits (feat/fix) going out at the same time.

how does that sound?